<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cashierpage</class>
 <widget class="QDialog" name="cashierpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>801</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="myapplogo">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>My App</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cashier_search">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Search </string>
   </property>
  </widget>
  <widget class="QComboBox" name="Cashier_combo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Names</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Categories</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Factories</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Cashier_in">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cashier_logout">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cashier_input_searchline">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>130</y>
     <width>811</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cashier_show_basket">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>180</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show The Bill</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cashier_searchbutton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_name_lable">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>47</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit :</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_credit_lable">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cashier_buyCredit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy Credit</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>721</width>
     <height>181</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Factory</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Category</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>License</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DisCount %</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>171</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>811</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>License of the Product :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cashier_li1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cashier_li2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cashier_li3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cashier_addBaslket">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to Bill</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>The Number of Purchases :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cashier_purchline">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>450</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment :</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_payment">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salebutton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>AUCTION</string>
   </property>
  </widget>
  <widget class="QLabel" name="alarm">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>21</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="alarm_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>47</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="digital_time">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cashier_input_searchline</tabstop>
  <tabstop>Cashier_combo</tabstop>
  <tabstop>cashier_searchbutton</tabstop>
  <tabstop>cashier_li1</tabstop>
  <tabstop>cashier_li2</tabstop>
  <tabstop>cashier_li3</tabstop>
  <tabstop>cashier_purchline</tabstop>
  <tabstop>cashier_addBaslket</tabstop>
  <tabstop>cashier_show_basket</tabstop>
  <tabstop>cashier_buyCredit</tabstop>
  <tabstop>cashier_logout</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
